lib/settings-panel.js

```diff
@@ -323,7 +323,10 @@ export default class SettingsPanel extends CollapsibleSectionPanel {
         return floatValue
       }
     } else if (type === 'array') {
-      let arrayValue = (value || '').split(',')
+      let arrayValue = (value || '').split(/[^\\],/)
```

Comment:
`.split(/[^\\],/)` will remove any character before a comma

```js
"1, 2".split(/[^\\],/) // ["", "2"]
```

I would still split on a comma and check if the last character in any group is a `\` then concatenate the groups

```suggestion
      let arrayValue = (value || '').split(',')
      arrayValue = arrayValue.reduce((values, val) => {
        const last = values.length - 1
        if (last >= 0 && values[last].endsWith("\\")) {
          values[last] = values[last].replace(/\\$/, ",") + val
        } else {
          values.push(val)
        }
        return values
      }, [])
```